TestCases/coolprop/coolprop.cfg

```diff
@@ -0,0 +1,246 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Non-ideal compressible fluid flow in a converging-         %
+%                   diverging supersonic nozzle with CoolProp library          %
+% Author: Peng Yan, Alberto Guardone                                           %
+% Institution: Politecnico di Milano                                           %
+% Date: 2022.10.8                                                              %
+% File Version  7.4.0 Blackbird                                                %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES,
+%                               FEM_EULER, FEM_NAVIER_STOKES, FEM_RANS, FEM_LES,
+%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
+%                               POISSON_EQUATION)
+SOLVER= RANS
+%
+% Specify turbulence model (NONE, SA, SA_NEG, SST, SA_E, SA_COMP, SA_E_COMP)
+KIND_TURB_MODEL= SST
```

Comment:
The reason is that these copy pastes become outdated easily, as shown here, I've never known SU2 with a POISSON_EQUATION :laughing:

Reply:
You are right, I also never used SU2 with a POISSON_EQUATION